nodejs/eks/examples/eks-hello-world/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "aws-typescript",
+    "devDependencies": {
+        "@types/node": "latest"
+    },
+    "dependencies": {
+        "@pulumi/aws": "latest",
```

Comment:
For examples, I'd be inclined to pin all deps, just in case.

Reply:
I _think_ we tend to prefer `latest` for the examples in these repos, because they double as canary tests for breaking changes. That said, @lukehoban or @CyrusNajmabadi would be able to say for sure. (I forget, and to my dismay, when I checked the AWS repo, I see several `dev` dependencies have crept in...)